data/mods/ssb/abilities.ts

```diff
@@ -10,6 +10,31 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 	},
 	*/
 	// Please keep abilites organized alphabetically based on staff member name!
+	guardianangel: {
```

Comment:
This is not what the ability is supposed to do, it is supposed to randomly choose 2 types the resist the weaknesses of the previous pokemon on the user's team. it is also supposed to modify the base stats of the pokemon to the pokemon that was before it.